src/partition/partition_op.h

```diff
@@ -70,6 +70,74 @@ IdArray MapToGlobalFromRemainder(
     IdArray local_idx,
     int part_id);
 
+/**
+ * @brief Create a permutation that groups indices by the part id.
```

Comment:
Can you give an example in the docstring?

Reply:
Example added in 
3c761c8